include/qpid/dispatch/internal/export.h

```diff
@@ -0,0 +1,27 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*   http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+/// @cond INTERNAL
+
+/// import/export macros
+#if defined(_WIN32)
```

Comment:
I suggest we remove any code that accommodates WIN32. In fact, I would recommend that we go thru the already existing Dispatch code and remove any code that has this kind of #if defined(_WIN32)
The reason I say this is that we don't want to have any code that even hints at anything to do with WIN32

Reply:
In this PR, I agree.

In the entire codebase, I think it is pointless, since there are (as far as I know only two instances: in the doctest.h framework, which we should not edit, and in CMakeLists.txt which is code "borrowed" from LLVM build scripts, which is also rather unprofitable to modify, in my view).

Reply:
> In the entire codebase, I think it is pointless, since there are (as far as I know only two instances: in the doctest.h framework, which we should not edit, and in CMakeLists.txt which is code "borrowed" from LLVM build scripts, which is also rather unprofitable to modify, in my view).

Make sense. Let's remove it from the PR. Thanks.

Reply:
Removed, thanks.